x-pack/plugins/cases/public/components/all_cases/utility_bar.tsx

```diff
@@ -159,14 +147,11 @@ export const CasesTableUtilityBar: FunctionComponent<Props> = ({
         </UtilityBarGroup>
       </UtilityBarSection>
       <ConfirmDeleteCaseModal
-        caseTitle={deleteThisCase.title}
+        caseTitle={deleteCases[0]?.title}
```

Comment:
I guess this is needed in case we have a bulk action with one case selected, right?